plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/app/PluginStrings.js

```diff
@@ -232,6 +232,9 @@ Ext.define('NX.coreui.app.PluginStrings', {
     ADMIN_BLOBSTORES_LIST_NEW_BUTTON: 'Create blobstore',
     ADMIN_BLOBSTORES_LIST_NAME_COLUMN: 'Name',
     ADMIN_BLOBSTORES_LIST_TYPE_COLUMN: 'Type',
+    ADMIN_BLOBSTORES_LIST_BLOB_COUNT_COLUMN: 'Blob Count',
```

Comment:
Correct. For the screenshots I tested by creating and uploading a single component to a nuget repo('default' blobstore) so that has 1, as well as creating an unpopulated blobstore through the UI